src/layouts/AllServices.test.tsx

```
@@ -0,0 +1,223 @@
+
+const mockUseFlag = jest.fn<(flagName: string) => boolean>();
+jest.mock('@unleash/proxy-client-react', () => ({
+  useFlag: (flagName: string) => mockUseFlag(flagName),
+}));
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider, createStore } from 'jotai';
+import AllServices from './AllServices';
+import { describe, it, expect, jest, beforeEach } from '@jest/globals';
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)